javascript/javascript1/week4/script.js

```diff
@@ -112,6 +113,17 @@ function getReply(command) {
                             return "Invalid operator";
                     }
                     return calculationResult;
+
+                case "timerMatch":
+                    // Parses the minutes from the match object and converts it to an integer
+                    const minutes = parseInt(match[1]);
+
+                    // Sets a timer to execute a callback function after the specified number of minutes
+                    setTimeout(() => {
+                        console.log("Timer done");
+                    }, minutes * 60 * 1000); // minutes * seconds * milliseconds
+
+                    return `Timer set for ${minutes} minutes`;
```

Comment:
Abstract away into separate function.

Also, for [extra reading](https://alonewolf24.medium.com/mastering-javascripts-settimeout-and-cleartimeout-functions-7eb63a369175) but definitely not a part of the homework is the importance of clearTimeout. You will likely experience odd behaviour in React when using setTimeout if you don't remember to clearTimeout when you're finished with the setTimeout function.

Reply:
Actually I try to solve it but I couldn't